src/timepicker/timepicker.spec.ts

```diff
@@ -0,0 +1,369 @@
+import {inject, async, addProviders} from '@angular/core/testing';
+import {TestComponentBuilder} from '@angular/compiler/testing';
+import {By} from '@angular/platform-browser';
+
+import {Component} from '@angular/core';
+
+import {NgbTimepicker} from './timepicker';
+
+function getInputs(el: HTMLElement) {
+  return el.querySelectorAll('input');
+}
+
+function getButtons(nativeEl: HTMLElement) {
+  return nativeEl.querySelectorAll('button.btn-link');
+}
+
+function getMeridianButton(nativeEl: HTMLElement) {
+  return nativeEl.querySelector('button.btn-primary-outline');
+}
+
+function expectToDisplayTime(el: HTMLElement, time: string) {
```

Comment:
Yes, we should move to Jasmine matchers in place of helper functions for most of our tests. I will open a separate issue for this.